templates/todo/projects/python-mongo/.repo/terraform/azure.yaml

```diff
@@ -0,0 +1,15 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/v1.0/azure.yaml.json
+
```

Comment:
Does this need something like:

`infra: terraform`

To opt into terraform?

Reply:
this will come within the terrraform provider pr.